<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="BodyLabel" name="BodyLabel_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TitleLabel" name="TitleLabel_6">
        <property name="text">
         <string>网页地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LineEdit" name="LineEdit_6"/>
      </item>
      <item>
       <widget class="TitleLabel" name="TitleLabel">
        <property name="text">
         <string>Electron版本(可选)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LineEdit" name="LineEdit"/>
      </item>
      <item>
       <widget class="TitleLabel" name="TitleLabel_2">
        <property name="text">
         <string>图标</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="BodyLabel" name="BodyLabel">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>未选择文件......</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="PushButton" name="PushButton">
          <property name="text">
           <string>选取文件</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="TitleLabel" name="TitleLabel_3">
        <property name="text">
         <string>应用名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LineEdit" name="LineEdit_3"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="TitleLabel" name="TitleLabel_4">
          <property name="text">
           <string>更新版本</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="PushButton" name="PushButton_3">
          <property name="text">
           <string>有问题？</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="LineEdit" name="LineEdit_4"/>
      </item>
      <item>
       <widget class="TitleLabel" name="TitleLabel_5">
        <property name="text">
         <string>应用加载时背景颜色</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="LineEdit" name="LineEdit_5"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="CheckBox" name="CheckBox">
        <property name="text">
         <string>始终置顶</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CheckBox" name="CheckBox_2">
        <property name="text">
         <string>全屏</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CheckBox" name="CheckBox_3">
        <property name="text">
         <string>禁用Dev Tools</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="CheckBox" name="CheckBox_4">
        <property name="text">
         <string>禁用右键</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="BodyLabel" name="BodyLabel_2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="SwitchButton" name="SwitchButton">
          <property name="text">
           <string>夜间模式</string>
          </property>
          <property name="onText">
           <string>夜间模式</string>
          </property>
          <property name="offText">
           <string>夜间模式</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ToolButton" name="ToolButton"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="IndeterminateProgressBar" name="IndeterminateProgressBar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="BodyLabel" name="BodyLabel_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PrimaryPushButton" name="PrimaryPushButton">
        <property name="text">
         <string>启动</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PlainTextEdit" name="PlainTextEdit">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">PlainTextEdit {
    color: green;
    background-color: black;
    border: 1px solid rgba(0, 0, 0, 13);
    border-bottom: 1px solid rgba(0, 0, 0, 100);
    border-radius: 5px;
    /* font: 14px &quot;Segoe UI&quot;, &quot;Microsoft YaHei&quot;; */
    padding: 0px 10px;
}
</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>等待唤醒......</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ToolButton</class>
   <extends>QToolButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>IndeterminateProgressBar</class>
   <extends>QProgressBar</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Main.qrc"/>
 </resources>
 <connections/>
</ui>
